javascripts/mods/ngp3/mastery_studies.js

```diff
@@ -289,7 +289,10 @@ var masteryStudies = {
 		303: "Meta Dimensions are stronger based on your galaxies.",
 		311: "Replicanti boost to all Infinity Dimensions is 17.3x stronger.",
 		312: "Meta-dimension boosts are 4.5% stronger and cost scale by 1 less.",
-		321: "Buff multiplier per 10 normal Dimensions to <span id='321effect'></span>x if it is 1x.",
+		321: function(){
+			s = (inQC(5) || inQC(7)) || (((inNC(13) && player.tickspeedBoosts == undefined) || player.currentChallenge == "postc1" || player.currentChallenge == "postcngm3_1") && player.galacticSacrifice != undefined)
+			return "Buff multiplier per 10 normal Dimensions to <span id='321effect'></span>x if it is 1x.<br>Currently:" +  s ? "active." : "inactive." 
```

Comment:
since the span with ID "321effect" is only defined by the return of a function, it can't be used as a target of getElementById elsewhere - this is breaking the game for me.

Reply:
more specifically this code is breaking:

`if (quantumed) document.getElementById("321effect").textContent=shortenCosts(new Decimal("1e430"))`

Uncaught TypeError: Cannot set property 'textContent' of null
    at updateMasteryStudyTextDisplay (mastery_studies.js:737)
    at updateMasteryStudyCosts (mastery_studies.js:437)
    at onLoad (load_functions.js:1930)
    at load_game (load_functions.js:2208)
    at initGame (game.js:7285)
    at onload ((index):19)